packages/manager/modules/billing/src/payment/method/add/views/legacyBillingAddress/translations/Messages_en_GB.json

```
@@ -4,5 +4,10 @@
   "billing_payment_method_add_bank_account_address_zip": "Postcode",
   "billing_payment_method_add_bank_account_address_city": "City",
   "billing_payment_method_add_bank_account_owner_name": "Surname and first name of owner or organisation owner",
-  "billing_payment_method_add_bank_account_owner_address": "Address of account owner"
+  "billing_payment_method_add_bank_account_owner_address": "Address of account owner",
+  "billing_payment_method_add_bank_account_owner_notification_info": "Within 24 to 48 hours, you will receive an email via the email address linked to your OVHcloud account. This email will contain a link for the electronic signature of your direct debit mandate.",
+  "billing_payment_method_add_bank_account_owner_info": "Warning: the bank account owner must also be the OVHcloud account owner.",
+  "billing_payment_method_add_bank_account_owner_confirmation_info": "Once your direct debit mandate has been received by our services, if it gets approved, you can then use this new payment method.",
+  "billing_payment_method_add_bank_account_data_process_by_ovh_info": "The data provided via this form is processed by OVHcloud as a controller to create and manage your SEPA direct debit. <a href=\" {{ dataProtectionOvhLink }} \" target=\" _blank \"> More about </a> how OVHcloud processes your data, and how to exercise your rights.",
+  "billing_payment_method_add_bank_account_data_process_by_yousign_info": "The creation of a SEPA direct debit requires to implement a SEPA mandate that we will ask you to execute by using an electronic signature solution provided by YOUSIGN a service provider dully authorized in accordance with “eIDAS” Regulation No. 910/2014 of 23 July 2014. In this context, the aforementioned data are communicated to YOUSIGN. <a href=\" {{ dataProtectionYouSignLink }} \" target=\" _blank \"> More about </a> how YOUSIGN processes your data."
```

nitpick(non-blocking): Looks like there some extra space.

```suggestion
  "billing_payment_method_add_bank_account_data_process_by_yousign_info": "The creation of a SEPA direct debit requires to implement a SEPA mandate that we will ask you to execute by using an electronic signature solution provided by YOUSIGN a service provider dully authorized in accordance with “eIDAS” Regulation No. 910/2014 of 23 July 2014. In this context, the aforementioned data are communicated to YOUSIGN. <a href=\" {{ dataProtectionYouSignLink }} \" target=\" _blank \">More about</a> how YOUSIGN processes your data."
```